cmd/sql/flow.go

```diff
@@ -317,6 +370,94 @@ func executeCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func executeDeployCmd(cmd *cobra.Command, args []string) error {
```

Comment:
nit: ideally I would expect most of the logic inside this function to be part of `sql` package instead of `cmd/sql`. In my mind this is how I see the SQL code structure at a high level:
`cmd/sql` -> anything related to command declaration for all the SQL CLI commands
`sql` -> all the business logic outside the `cobra` command declaration for all the SQL CLI commands